image-rs/src/image.rs

```diff
@@ -526,7 +526,7 @@ mod tests {
             // Azure Container Registry
             "mcr.microsoft.com/hello-world",
             // Docker container Registry
-            "docker.io/i386/busybox",
+            "docker.io/busybox",
```

Comment:
This is from my earlier commit

Reply:
Its cherry picked, so you remain author. Had some github with that particular rebase but now its all fine.

Reply:
Fixed git history now